Dockerfile.unittesting.Dockerfile

```diff
@@ -26,7 +26,10 @@ FROM artifactory.algol60.net/docker.io/library/golang:1.16-alpine
 
 RUN set -ex \
     && apk -U upgrade \
-    && apk add build-base
+    && apk add build-base \
+    && apk add bash \
```

Comment:
why are you adding this into the unit testing?

Reply:
All of these additions are there in order to run the wait-for-discovery.sh script.   I wanted bash so
the script would be easier to write.   curl and jq are used to fetch info from HSM and parse it in 
order to see if discovery is complete.